src/main/java/com/duke/util/Ui.java

```diff
@@ -0,0 +1,209 @@
+package com.duke.util;
+
+import com.duke.task.Task;
+import com.duke.task.TaskList;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Represents the handler of inputs and outputs of the user interface.
+ */
+public class Ui {
+    private String space = "     ";
+    private String line = space + "_____________________________________________";
+    Scanner scanner = new Scanner(System.in);
+
+    /**
+     * Prints the error message to the user to communicate that
+     * there is a problem loading data from storage.
+     */
+    public void showLoadingError() {
+        System.out.println(space + "OOPS! There is a problem loading save file");
+    }
+
+    public String getLoadingErrorMessage() {
+        return space + "OOPS! There is a problem loading save file";
+    }
+
+    /**
+     * Prints the welcome message when Duke starts up.
+     */
+    public void showWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke" + "\n" + space + "What can I do for you?");
+
+    }
+
+    public String getWelcomeMessage() {
+        String logo = " ____        _        \n"
```

Comment:
Can make extend the variable "line" and "logo" to the class, so that you do not have to repeat yourself in code. Also, if you want to change the logo later, you can simply change it in a single location.